model/task/db.go

```diff
@@ -2839,6 +2840,16 @@ func SetHasAnnotations(taskId string, execution int) error {
 	return errors.Wrapf(err, "marking task '%s' as having annotations", taskId)
 }
 
+// SetNumNextTaskDispatches sets the next task count for a task.
```

Comment:
nit: `sets the number of times a host has requested this task and execution as its next task`.